packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -82,15 +82,20 @@ def get_measurement_service_stub(
     return v2_measurement_service_pb2_grpc.MeasurementServiceStub(channel)
 
 
-def get_all_registered_measurement_service_classes(discovery_client: DiscoveryClient) -> List[str]:
+def get_all_registered_measurement_info(
+    discovery_client: DiscoveryClient,
+) -> Tuple[List[str], List[str]]:
     """Returns the service classes of all the registered measurement services."""
```

Comment:
doc needs to update as it is returning the display name as well along with the service class.